man/docker-run.1.md

```diff
@@ -371,6 +372,9 @@ its root filesystem mounted as read only prohibiting any writes.
 **--sig-proxy**=*true*|*false*
    Proxy received signals to the process (non-TTY mode only). SIGCHLD, SIGSTOP, and SIGKILL are not proxied. The default is *true*.
 
+**--memory-swappiness**=*60*
+   Tuning the memory swappiness of container. The value in the range of 0-100 is accepted.
```

Comment:
Tune a container's memory swappiness behavior. Accepts an integer between 0 to 100.